staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -318,22 +318,22 @@ func chooseIPFromHostInterfaces(nw networkInterfacer, addressFamilies AddressFam
 		return nil, fmt.Errorf("no interfaces found on host.")
 	}
 	for _, family := range addressFamilies {
-		klog.V(4).Infof("Looking for system interface with a global IPv%d address", uint(family))
+		klog.V(4).InfoS("Looking for system interface with a global address", "ipVersion", uint(family))
```

Comment:
The InfoS output is hard to understand. I don't think a reader will understand what the `ipVersion` variable is referring to.